bookshelf/gce/startup-script.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START all]
+set -e
+
+# Talk to the metadata server to get the project id
+PROJECTID=$(curl -s "http://metadata.google.internal/computeMetadata/v1/project/project-id" -H "Metadata-Flavor: Google")
+
+# Get the source code
+export HOME=/root
+git config --global credential.helper gcloud.sh
+
+# Change branch from master if not using master
+git clone https://source.developers.google.com/p/$PROJECTID /opt/app -b add-compute-engine
+
+# [START logging]
+curl -s "https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh" | bash
+cat >/etc/google-fluentd/config.d/bookshelf.conf << EOF
+<source>
+  type tail
+  format none
+  path /opt/app/log/*.log
+  pos_file /var/tmp/fluentd.app.pos
+  read_from_head true
+  rotate_wait 10s
+  tag bookshelf
+</source>
+EOF
+service google-fluentd restart &
+# [END logging]
+
+# [START php]
+# Install PHP and dependencies from apt
+apt-get update
+apt-get install -y php5 nginx build-essential php5-fpm # libpcre3 libpcre3-dev zlib1g-dev
+
+# Install composer
+curl -sS https://getcomposer.org/installer | \
+    /usr/bin/php -- \
+    --install-dir=/usr/local/bin \
+    --filename=composer
+
+# run composer
+composer install -d /opt/app/bookshelf --no-ansi --no-progress
+# [END php]
+
+# [START nginx]
+# disable the default nginx configuration
+rm /etc/nginx/sites-enabled/default
+
+# enable our nginx configuration
+cat /opt/app/bookshelf/gce/nginx/bookshelf.conf > /etc/nginx/sites-available/bookshelf.conf
```

Comment:
I saw it done [here](https://github.com/GoogleCloudPlatform/getting-started-ruby/blob/master/7-compute-engine/gce/configure.sh#L50) and decided to stay consistent